github/MainClass.py

```diff
@@ -253,7 +253,7 @@ def get_events(self):
             github.Event.Event, self.__requester, "/events", None
         )
 
-    def get_user(self, login=github.GithubObject.NotSet):
+    def get_user(self, login="{0}".format(github.GithubObject.NotSet)):
```

Comment:
I daresay this isn't the correct place, and we should cast login to str on line 269.